### PR TITLE
gh-118181: Fix parameter markup in AST docs

### PR DESCRIPTION
**Pull request by tomasr8**
There were quite a few so I hope I got all of them :sweat_smile: 
Also, this is a minor docs fix so I'm guessing no news entry is required?

cc @Eclips4 :)

* Issue: gh-118181

----
📚 Documentation preview 📚: https://cpython-previews--124473.org.readthedocs.build/

### CODE REVIEW

File: Doc/library/ast.rst [hunk 1]
```diff
@@ -267,7 +267,7 @@ Literals
 
 .. class:: Constant(value)
 
-   A constant value. The ``value`` attribute of the ``Constant`` literal contains the
+   A constant value. The *value* attribute of the ``Constant`` literal contains the
```
Comment (JelleZijlstra) [inline Doc/library/ast.rst]: Looking at this now, I'm not sure it's actually right (sorry for the trouble!). We mark up parameters with *italics*, but attributes with double backticks. These names are both parameters to the constructor and attributes on the resulting object, but the attribute use case is the most important, as it's more common to inspect an AST than to create AST nodes.

I'll ask on the docs Discord for more opinions.

Comment (tomasr8) [reply Doc/library/ast.rst]: Based on the discussion on Discord, I changed the (few) cases that used `*value*` to use backticks instead :)

**Review by Eclips4 — APPROVED** [review]
Thanks, Tomas! I would like to believe that I didn't miss anything while reviewing it.

**Review by JelleZijlstra — APPROVED** [review]

**Comment by JelleZijlstra** [comment]
Thanks @tomasr8! Are you interested in doing the 3.12 backport? I guess it failed because I added some AST nodes in 3.13.

**Comment by tomasr8** [comment]
> Thanks @tomasr8! Are you interested in doing the 3.12 backport? I guess it failed because I added some AST nodes in 3.13.

Sure! I can have a look later today or tomorrow if it's ok? :)

**Comment by JelleZijlstra** [comment]
Thanks, there's no hurry!

**Pull request merged**